docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -226,17 +247,119 @@ yugabyted configure --fault_tolerance=zone
 --base_dir *base-directory*
 : The base directory for the yugabyted server.
 
+--log_dir *log-directory*
+: The directory to store yugabyted logs. Must be an absolute path. This flag controls where the logs of the YugabyteDB nodes are stored.
+
+
+#### encrypt_at_rest
+
+Use the `yugabyted configure encrypt_at_rest` subcommand to enable or disable encryption at rest for the deployed cluster.
+
+#### Syntax
+
+```text
+Usage: yugabyted configure encrypt_at_rest [flags]
+```
+
+Examples:
+- To enable encryption at rest for a deployed YugabyteDB cluster, execute the following:
+
+  ```sh
+  yugabyted configure encrypt_at_rest --enable
+  ```
+
+- To disable encryption at rest for a YugabyteDB cluster which has encryption at rest enabled, execute the following:
+
+  ```sh
+  yugabyted configure encrypt_at_rest --disable
+  ```
+
+#### Flags
+
+-h | --help
+: Print the command line help and exit.
+
+--disable *disable*
+: Disable encryption at rest for the cluster. No need to set a value for the flag. Use --enable or --disable flag to toggle encryption features on a YugabyteDB cluster.
+
+--enable *enable*
+: Enable encryption at rest for the cluster. No need to set a value for the flag. Use --enable or --disable flag to toggle encryption features on a YugabyteDB cluster.
```

Comment:
Enable encryption at rest for the cluster. There is no need to set a value for the flag. Use `--enable` or `--disable` flag to toggle encryption features on a YugabyteDB cluster.